Add types for isValidAddress

When using ripple-lib in a TypeScript project with `strict` mode enabled, I get this error:

```
node_modules/ripple-lib/dist/npm/common/schema-validator.d.ts:1:32 - error TS7016: Could not find a declaration file for module 'ripple-address-codec'. '/Users/kincaid/ilp/settlement-xrp/node_modules/ripple-address-codec/src/index.js' implicitly has an 'any' type.
  Try `npm install @types/ripple-address-codec` if it exists or add a new declaration (.d.ts) file containing `declare module 'ripple-address-codec';`

1 import { isValidAddress } from 'ripple-address-codec';
                                 ~~~~~~~~~~~~~~~~~~~~~~
```

Adding the suggested type declaration file fixes it, but it'd be nice if that wasn't required and/or `isValidAddress` was strongly typed (right now it's just `any`).

--- a/docs/index.md
+++ b/docs/index.md
@@ -91,6 +91,7 @@
   - [xrpToDrops](#xrptodrops)
   - [dropsToXrp](#dropstoxrp)
   - [iso8601ToRippleTime](#iso8601torippletime)
+  - [rippleTimeToISO8601](#rippletimetoiso8601)
   - [txFlags](#txflags)
   - [schemaValidator](#schemavalidator)
   - [schemaValidate](#schemavalidate)
@@ -5858,6 +5859,34 @@ api.iso8601ToRippleTime('2017-02-17T15:04:57Z');
 
 ```json
 540659097
+```
+
+## rippleTimeToISO8601
+
+`rippleTimeToISO8601(rippleTime: number): string`
+
+This method takes the number of seconds since the "Ripple Epoch" of January 1, 2000 (00:00 UTC) and returns a string representation of a date.
+
+The Ripple Epoch is 946684800 seconds after the Unix Epoch.
+
+This method is useful for interpreting timestamps returned by the rippled APIs. The rippled APIs represent time as an unsigned integer of the number of seconds since the Ripple Epoch.
+
+### Parameters
+
+`rippleTime`: A number of seconds since the Ripple Epoch.
+
+### Return Value
+
+A string representing a date and time, created by calling a `Date` object's `toISOString()` method.
+
+### Example
+
+```javascript
+api.rippleTimeToISO8601(540659097);
+```
+
+```json
+'2017-02-17T15:04:57.000Z'
 ```
 
 ## txFlags

--- a/docs/src/index.md.ejs
+++ b/docs/src/index.md.ejs
@@ -62,6 +62,7 @@
 <% include computeLedgerHash.md.ejs %>
 <% include xrpToDropsAndDropsToXrp.md.ejs %>
 <% include iso8601ToRippleTime.md.ejs %>
+<% include rippleTimeToISO8601.md.ejs %>
 <% include txFlags.md.ejs %>
 <% include schemaValidator.md.ejs %>
 <% include events.md.ejs %>

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "https-proxy-agent": "2.2.1",
     "jsonschema": "1.2.2",
     "lodash": "^4.17.4",
-    "ripple-address-codec": "^3.0.0",
+    "ripple-address-codec": "^3.0.3",
     "ripple-binary-codec": "0.2.2",
     "ripple-hashes": "0.3.2",
     "ripple-keypairs": "^0.10.1",

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "https-proxy-agent": "2.2.1",
     "jsonschema": "1.2.2",
     "lodash": "^4.17.4",
-    "ripple-address-codec": "^2.0.1",
+    "ripple-address-codec": "^3.0.0",
     "ripple-binary-codec": "0.2.2",
     "ripple-hashes": "0.3.2",
     "ripple-keypairs": "^0.10.1",

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "https-proxy-agent": "2.2.1",
     "jsonschema": "1.2.2",
     "lodash": "^4.17.4",
-    "ripple-address-codec": "^3.0.3",
+    "ripple-address-codec": "^3.0.4",
     "ripple-binary-codec": "0.2.2",
     "ripple-hashes": "0.3.2",
     "ripple-keypairs": "^0.10.1",

--- a/src/api.ts
+++ b/src/api.ts
@@ -5,6 +5,7 @@ import {
   validate,
   xrpToDrops,
   dropsToXrp,
+  rippleTimeToISO8601,
   iso8601ToRippleTime,
   txFlags
 } from './common'
@@ -337,6 +338,7 @@ class RippleAPI extends EventEmitter {
 
   xrpToDrops = xrpToDrops
   dropsToXrp = dropsToXrp
+  rippleTimeToISO8601 = rippleTimeToISO8601
   iso8601ToRippleTime = iso8601ToRippleTime
   txFlags = txFlags
 

